db/migrate/20201208093803_change_activity_programme_status_to_enum.rb

```diff
@@ -0,0 +1,25 @@
+class ChangeActivityProgrammeStatusToEnum < ActiveRecord::Migration[6.0]
+  def up
+    add_column :activities, :programme_status_integer, :integer
+
+    Activity.all.each do |activity|
```

Comment:
This should probably be `find_each` which avoids loading every Activity into memory at once.